guides/common/modules/proc_using-novnc-to-access-virtual-machines.adoc

```diff
@@ -13,9 +13,16 @@ You can use your browser to access the VNC console of VMs created by {Project}.
 .Prerequisites
 * You must have a virtual machine created by {Project}.
 * For existing virtual machines, ensure that the *Display type* in the *Compute Resource* settings is *VNC*.
+ifdef::katello,orcharhino,satellite[]
 * You must import the Katello root CA certificate into your {ProjectServer}.
 Adding a security exception in the browser is not enough for using noVNC.
 For more information, see {ConfiguringUserAuthenticationDocURL}importing-the-katello-root-ca-certificate[Installing the Katello root CA certificate] in _{ConfiguringUserAuthenticationDocTitle}_.
+endif::[]
+ifdef::foreman-deb,foreman-el[]
+* Import the CA certificate that is set as `websockets_ssl_cert`.
+Adding a security exception in the browser is not enough for using noVNC.
+You can get the path by running `{foreman-installer} --full-help | grep -- "--foreman-websockets-ssl-cert\|--foreman-server-ssl-cert"`.
```

Comment:
Please look up Foreman settings via the UI, API or Hammer, not by using the installer. In this particular case the settings is under the Authentication section, but users can also search for it.

However, if you take my above suggestion to simplify it then this sentence can be dropped.

Reply:
I added this as alternative for foreman-el and foreman-deb.